frontend/packages/ux-editor/src/components/TextResource/hooks/useTextResourceValue.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { useTextResourceValue } from './useTextResourceValue';
+import { textResourceByLanguageAndIdSelector } from '@altinn/ux-editor/selectors/textResourceSelectors';
+import { DEFAULT_LANGUAGE } from 'app-shared/constants';
+import { renderHook } from '@testing-library/react';
+import { ServicesContextProvider } from 'app-shared/contexts/ServicesContext';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+import React from 'react';
+import { queriesMock } from 'app-shared/mocks/queriesMock';
+
+// Test data
+const id = 'testId';
+const value = 'testValue';
+
+// Mocks
+jest.mock('@altinn/ux-editor/hooks', () => ({
+  useTextResourcesSelector: jest.fn((selector) => ({
+    value: selector === textResourceByLanguageAndIdSelector(DEFAULT_LANGUAGE, id) ? value : value,
+  })),
+}));
```

Comment:
Its not necessary to mock this hook. 
It uses the data from the textResourceQuery and textLangaugesQuery so we can simply put it on the cache 🫡

Reply:
Fixed